### ISSUE TITLE
raidboss : UCOB DragonMarks callouts always call EN

### ISSUE THREAD

**Issue by Una-P** [quisquous/cactbot#1434]
First of all, sorry for my awful English.
Yesterday I played UCOB nael phase, and detected a small bug but I can't fix it with my skill.

At <a href="https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js#L1033">UCU Nael Dragon Placement</a>,
https://github.com/quisquous/cactbot/blob/e27c33a34f69e5ba53ac4937e7a9468b57d78f29/ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js#L1052
That callout supposed to look like `징:11시,6시,9시(넓음)`, but it works like `징: NW,S,W(넓음)`.
and also <a href="https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js#L1057">UCU Nael Dragon Dive Marker Me</a> shows same problem, too.

I think problem is <a href="https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js#L1011">this code's something</a>, array has to be different depending on user's language, but it always call `en: ['N', 'NE', 'E', 'SE', 'S', 'SW', 'W', 'NW'], `array.
so I changed to `en: ['12시', '1시', '3시', '5시', '6시', '7시', '9시', '11시'],` it works well.

**Comment by JLGarber** [comment]
That's very odd. [Line 1020](https://github.com/quisquous/cactbot/blob/e27c33a34f69e5ba53ac4937e7a9468b57d78f29/ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.js#L1020) should force that to be KO. I wonder whether there's a disconnect between the client language and the selected locale in cactbot?

**Comment by Una-P** [comment]
Can you tell me where is locale selection button? I've never know that but it's no matter when I used cactbot...

**Comment by quisquous** [comment]
In this case, the language is the parser language.  It's what's selected from Plugins -> FFXIV Settings -> Game Language.

I thought the same thing as @JLGarber reading that code.  Nothing looks awry there.  Maybe it would be worth adding a `console.log('lang: ' + data.lang);` line to that function and seeing if that's the issue or if there's some logic error we're both missing?

**Comment by Una-P** [comment]
Thanks. here's my config 
> ACT Game language : EN
> my client language : EN
> Cactbot alarm and timeline language : KO

And ucob.js, I didn't change anything from 0.17.2 release. (except `ko: dir + '으로 이동',` I pulled #1426 )
I pulled my ucob js file at forked repo and compare current one, but there was nothing else strange about it.
I played yesterday, and can find same thing.
![1](https://user-images.githubusercontent.com/39465903/82135947-466cfd00-9843-11ea-9f87-26eab2a9dd47.PNG)
![2](https://user-images.githubusercontent.com/39465903/82135948-4836c080-9843-11ea-82d4-4dd7860f3e16.PNG)

**Comment by Una-P** [comment]
I'll replace it with the latest file and play tomorrow.

**Comment by quisquous** [comment]
Oh! I see what's happening.  `data.lang` is English, because that's what your parser language is.  That's why it displays in English.

I think now that #1400 has landed, maybe a followup is to change the uses of `data.lang` into `data.parserLang` and `data.displayLang`?.  I think there are cases where it is used for both, unfortunately.

**Comment by Una-P** [comment]
hmm.. I think i understand half.
This issue is solved at #1400, so if I change cactbot after #1400 version,  it will be fine, right?

**Comment by Una-P** [comment]
Anyway, Thank you for your help. I'll try at 2 hours later and close issue or give you another comment.
I hope it works well :)

**Comment by quisquous** [comment]
Sorry for the misunderstanding.  #1400 is a prerequisite that makes it possible to fix this bug with another pull request.  This bug is still not fixed.
